src/lib.rs

```diff
@@ -733,6 +797,51 @@ impl Certificate {
     pub fn stake_pool_registration(pool_registration: PoolRegistration) -> Certificate {
         certificate::Certificate::PoolRegistration(pool_registration.0).into()
     }
+
+    /// Create a Certificate for PoolRetirement
+    pub fn stake_pool_retirement(pool_retirement: PoolRetirement) -> Certificate {
+        certificate::Certificate::PoolRetirement(pool_retirement.0).into()
+    }
+
+    pub fn get_type(&self) -> String {
```

Comment:
Based on Account's usage of type as a string. Could be changed to an enum if preferred.

Reply:
I would prefer an enum that way we can statically generate an error in Yoroi if this function ever changes